tox.ini

```diff
@@ -33,6 +34,7 @@ deps =
 	pytest-xdist
 	py27: mock
 	requests
+	jwcrypto
 
 [testenv:py37-docs]
 basepython = python
```

Comment:
I believe you need to add jwcrypto to deps here as well to resolve `tox -e py37-docs` failure.

Reply:
> And, of course, failing tests have to be resolved. They are mostly isort/flake8 issues because I undid the disabling of those tests in #798. They should be really easy to fix:

Those things were fixed here https://github.com/wiliamsouza/django-oauth-toolkit/pull/8. After merging, will rest only documentation and examples to merge this PR